src/server/game/Maps/Map.h

```diff
@@ -408,8 +408,8 @@ class Map : public GridRefManager<NGridType>
     void AddObjectToSwitchList(WorldObject* obj, bool on);
     virtual void DelayedUpdate(const uint32 diff);
 
-    //void UpdateObjectVisibility(WorldObject* obj, Cell cell, CellCoord cellpair);
-    //void UpdateObjectsVisibilityFor(Player* player, Cell cell, CellCoord cellpair);
+    void UpdateObjectVisibility(WorldObject* obj, Cell cell, CellCoord cellpair);
```

Comment:
I doubt this PR will ever work with just a "copy/paste" since the **UpdateObjectVisibility** and **UpdateObjectsVisibilityFor** are not used within our code.

You should check where TC uses those function but also please, double check the PR code to spot other similar issues if any

Reply:
I think it's not related, it works locally ;-)